lib/normalizers/v1.js

```diff
@@ -55,17 +55,22 @@ _.assign(Builders.prototype, {
      */
     auth: function (entityV1, options) {
         if (!entityV1) { return; }
+
+        var auth,
+            params,
+            mapper,
+            currentHelper,
+            helperAttributes,
+            prioritizeV2 = this.options.prioritizeV2;
+
         // Handle the noauth/normal case for new and legacy attributes first, in that order
-        if (util.notLegacy(entityV1, 'auth')) {
+        if ((util.notLegacy(entityV1, 'auth') || prioritizeV2) && (entityV1.auth || !prioritizeV2)) {
```

Comment:
This particular condition is completely unreadable. This could have been,

`if ((entityV1.auth && prioritizeV2) || util.notLegacy(entityV1, 'auth'))`